spacy/matcher/matcher.pyx

```diff
@@ -796,9 +796,13 @@ def _get_attr_values(spec, string_store):
                 attr = "ORTH"
             if attr == "IS_SENT_START":
                 attr = "SENT_START"
+                ent_iob = True
```

Comment:
```suggestion
```

(You can commit the suggestion directly from the comment if you want to. Sometimes this is easiest for minor edits.)